Library/Homebrew/brew.sh

```diff
@@ -772,6 +772,26 @@ To turn developer mode off, run $(bold "brew developer off")
   export HOMEBREW_DEV_CMD_RUN="1"
 fi
 
+if [[ "${HOMEBREW_COMMAND}" == "audit" ]] ||
```

Comment:
I was about to suggest `%w(audit bottle bump-cask-pr ...).each { |cmd| ... }`, then I realised this is bash. 🪦